### ISSUE TITLE
[STUDIO 1.1.0 beta] Reflet Courbe de chauffe et REDY 14.2.1

### ISSUE THREAD

**Issue by WITNicolas** [witsa/synapps#448]
Le reflet Courbe de chauffe n'est pas rafraichi malgré la validation.
Il faut faire un refresh de toutes la page pour que les données soient affichées.
Par contre les actions sont bien réalisées en écriture, donc c'est uniquement le visuel de la courbe qui ne bouge pas.

![image](https://user-images.githubusercontent.com/33632664/160701824-48537571-6e73-4450-8aae-a4f7392b9b38.png)

**Comment by WITNicolas** [comment]
En il y a un soucis de fonctionnement. En absculant en mode réduit dans le REDY, le visuel ne suit pas.
![image](https://user-images.githubusercontent.com/33632664/160704336-b06b2273-6fe1-4204-ad42-00df4580d37d.png)

**Comment by manufitoussiwit** [comment]
> En il y a un soucis de fonctionnement. En absculant en mode réduit dans le REDY, le visuel ne suit pas. ![image](https://user-images.githubusercontent.com/33632664/160704336-b06b2273-6fe1-4204-ad42-00df4580d37d.png)

Effectivement, il a un pb dans ce cas : mauvais calcul de la part de synapps. Je vais corriger ça dans la version 2.2.1 du runtime et ajouter un patch.

**Comment by manufitoussiwit** [comment]
Pour le premier pb, je vais essayer de reproduire.

**Comment by manufitoussiwit** [comment]
Je n'arrive pas a reproduire. Il est possible que le reflet soit en mode "édition en cours" et dans ce cas, il ne se raffraichit pas tant qu'aucune action n'est faire. Je vais d'ailleurs modifier un peu le reflet afin d'afficher mieux les boutons d'actions. Par contre, pourquoi le reflet est ne mode "edition en cours", ça je sais pas. 
Appelle moi quand tu peux pour qu'on regarde ensemble.

**Comment by manufitoussiwit** [comment]
Eventuellement, ajoute ce composite à coté de ton reflet : 

```text
SYNAPPS-STUDIO-ACTOR|{"type":"display/html","key":"reflect-style","properties":{"content":"<style>\n  .redy-reflect-details .content p.text-center.text-nowrap.show-slow.mt-1,\n  .redy-reflect-details .content .actions {\n    position: sticky;\n    bottom: 0;\n    background-color: transparent;\n    margin: 0 auto;\n    width: fit-content;\n  }\n\n  .redy-reflect-details .content .reflect-details-heating-curve canvas {\n    max-height: 10em !important;\n    width: 14em !important;\n    margin: 0 auto;\n  }\n</style>"}}
```

Il va permettre de placer les boutons toujours visible et réduire la taille du graphique. C'est une amélioration que je vais rajouter aussi dans la version 2.3.1 du runtime et patché aussi.

**Comment by manufitoussiwit** [comment]
hum, j'avais pas compris en fait, c'est le graphique qui est pas à jour.
